### PR TITLE
Remove NewChat page; Fix two tab chat close issue

### LINKED ISSUE #1199: [two tab issue] Implement a 'watcher' that will always clear messages in the case when the widget has been minimized.
In all cases, when the visitor has ended the chat and minimized the widget, we want the next action, when the visitor clicks the icon again, to start a new session.  

Currently in the two tab case, one of the tabs can easily get out of sync.

### PR DESCRIPTION
**Pull request by bhardwajaditya**
Closes: https://github.com/WideChat/Rocket.Chat/issues/1199

### CODE REVIEW

**Comment by ear-dev** [comment]
@bhardwajaditya @Shailesh351 I'm wondering if we should maybe combine this PR with this one https://github.com/WideChat/Rocket.Chat.Livechat/pull/145

At the very least it will make testing easier for me, and perhaps they are related enough?  What do you guys think?

**Comment by Shailesh351** [comment]
Yeah I think it's fine @ear-dev

**Comment by bhardwajaditya** [comment]
@ear-dev merged this one with the new chat page remove PR

**Comment by ear-dev** [comment]
Thanks @bhardwajaditya .....  I'll test.

**Comment by ear-dev** [comment]
This works great!  thanks @bhardwajaditya 

@Shailesh351 can you do a quick review please?  Thanks....

File: src/routes/ChatFinished/container.js [hunk 1]
```diff
@@ -11,9 +11,7 @@ export class ChatFinishedContainer extends Component {
 		route('/');
 	}
 
-	render = (props) => (
-		<ChatFinished {...props} onRedirectChat={this.handleRedirect} />
-	)
+	render = (props) => <ChatFinished {...props} onRedirectChat={this.handleRedirect} />
```
Comment (Shailesh351) [inline src/routes/ChatFinished/container.js]: I think this change is not needed. Can you revert it, please? Other things look good.

Comment (bhardwajaditya) [reply src/routes/ChatFinished/container.js]: Fixed it

**Review by ear-dev — APPROVED** [review]

**Pull request merged**